test/units/testsuite-82.sh

```diff
@@ -139,8 +139,8 @@ EOF
     # This sets DefaultDependencies=no so that it remains running until the
     # very end, and IgnoreOnIsolate=yes so that it isn't stopped via the
     # "testsuite.target" isolation we do on next boot
-    systemd-run -p Type=notify -p DefaultDependencies=no -p IgnoreOnIsolate=yes --unit=testsuite-82-survive.service "$T"
-    systemd-run -p Type=exec -p DefaultDependencies=no -p IgnoreOnIsolate=yes --unit=testsuite-82-nosurvive.service sleep infinity
+    systemd-run -p Type=notify -p SurviveSoftReboot=yes --unit=testsuite-82-survive.service "$T"
+    systemd-run -p Type=exec -p SurviveSoftReboot=yes --unit=testsuite-82-nosurvive.service sleep infinity
```

Comment:
The comment is outdated now. But as I wrote above, I don't think SurviveSoftReboot/IgnoreOnSoftReboot=yes should imply IgnoreOnIsolate=yes.